src/styles/index.scss

```diff
@@ -1,3 +1,104 @@
+@import './utils/mixins';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
[checklist:](https://github.com/mate-academy/layout_product-cards?tab=readme-ov-file#checklist) there 2 BEM blocks card and stars each in it own file
create separate file for each BEM block